src/platform/Linux/CHIPPlatformConfig.h

```diff
@@ -57,7 +57,7 @@ using CHIP_CONFIG_PERSISTED_STORAGE_KEY_TYPE = const char *;
 #endif // CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS
 
 #ifndef CHIP_LOG_FILTERING
-#define CHIP_LOG_FILTERING 0
+#define CHIP_LOG_FILTERING 1
```

Comment:
this would enable log filtering for all linux example/app, can we just enable it for python only?

Reply:
maybe defined from controller/python/BUILD.gn?

Reply:
I was thinking the same at first, I tried by adding a define in `src/controller/python/BUILD.gn`:

```gn
  defines = [ "CHIP_LOG_FILTERING=1" ]
```

However, that failed at compile time:

```
FAILED: obj/src/controller/python/chip/_ChipDeviceCtrl.so obj/src/controller/python/chip/_ChipDeviceCtrl.map                                                                                                                                                                                                                  
g++ -O0 -fPIC -Werror -Wl,--fatal-warnings -Wl,-z,defs -fdiagnostics-color -Wl,--gc-sections -Wl,-Map,obj/src/controller/python/chip/_ChipDeviceCtrl.map -Wl,--cref @obj/src/controller/python/chip/_ChipDeviceCtrl.so.rsp -o obj/src/controller/python/chip/_ChipDeviceCtrl.so -shared                                       
/usr/bin/ld: obj/src/controller/python/_ChipDeviceCtrl.ChipDeviceController-ScriptBinding.cpp.o: in function `pychip_DeviceController_GetLogFilter':                                                                                                                                                                          
/home/sag/projects/project-chip/connectedhomeip-v1.2/out/python_lib/../../src/controller/python/ChipDeviceController-ScriptBinding.cpp:357:(.text.pychip_DeviceController_GetLogFilter+0x5): undefined reference to `chip::Logging::GetLogFilter()'                                                                           
/usr/bin/ld: obj/src/controller/python/_ChipDeviceCtrl.ChipDeviceController-ScriptBinding.cpp.o: in function `pychip_DeviceController_SetLogFilter':                                                                                                                                                                          
/home/sag/projects/project-chip/connectedhomeip-v1.2/out/python_lib/../../src/controller/python/ChipDeviceController-ScriptBinding.cpp:366:(.text.pychip_DeviceController_SetLogFilter+0x14): undefined reference to `chip::Logging::SetLogFilter(unsigned char)'  
```

But then I thought why bother, filtering shouldn't harm if not used, and the additional size/overhead is probably negligible for typical Linux systems :man_shrugging:

Reply:
yep, the macro cannot pass across gn component, i think we could enable this at default for linux, thanks